compilation error when including @linode/api-v4 as a dependency

when compiling with typescript we get the following error

```
node_modules/@linode/api-v4/lib/support/types.d.ts:1:24 - error TS2307: Cannot find module 'src/account/types' or its corresponding type declarations.

1 import { Entity } from 'src/account/types';
                         ~~~~~~~~~~~~~~~~~~~
```

workaround:

in a file called global.d.ts:
```ts
declare module "src/account/types" {
    export interface Entity {
    }
}
```

but this is not ideal!

--- a/packages/api-v4/src/support/types.ts
+++ b/packages/api-v4/src/support/types.ts
@@ -1,4 +1,5 @@
-import { Entity } from 'src/account/types';
+import { Entity } from '../account/types';
+
 export interface SupportTicket {
   opened: string;
   id: number;

